spec/controllers/admin/reports/actions_controller_spec.rb

```diff
@@ -146,13 +146,13 @@
       end
     end
 
-    context 'with Action as submit button' do
+    context 'with action as submit button' do
       subject { post :create, params: common_params.merge({ action => '' }) }
 
       it_behaves_like 'all action types'
     end
 
-    context 'with Action as submit button' do
+    context 'with moderation action as submit button' do
```

Comment:
ah, I think this should rather read:
```suggestion
    context 'with moderation action as an extra field' do
```

The difference is wrt. the action being defined by an `<input type="submit" name="action" />` vs `<input type="hidden" name="moderation_action" value="action" />`